scss/search.scss

```diff
@@ -0,0 +1,88 @@
+@import "./variables.scss";
```

Comment:
You don't need to import variables since you are importing this scss to main.scss and in main.scss variables are already imported. 
This caused 
@import url("https://fonts.googleapis.com/css2?family=Fira+Sans:ital,wght@0,400;0,500;0,700;1,400&family=Mukta:wght@300;400;700&display=swap");
to be imported many times in main.css.